src/UnityExtension/Assets/Editor/GitHub.Unity/UI/GitPathView.cs

```diff
@@ -120,7 +120,7 @@ public override void OnGUI()
                         CheckEnteredGitPath();
 
                         new FindExecTask("git", Manager.CancellationToken)
-                            .Configure(Manager.ProcessManager)
+                            .Configure(Manager.ProcessManager, false, true)
```

Comment:
This should just be `.Configure(Manager.ProcessManager, dontSetupGit: true)` for readability and future-proofing.